debian/polygon2/DEBIAN/md5sums

```diff
@@ -0,0 +1,9 @@
+98bcd2c2cf5bdbefce8ee5140873c268  usr/lib/python2.7/dist-packages/Polygon/IO.py
```

Comment:
and you should not commit the DEBIAN folder :)

Reply:
Duly